spot_rl_experiments/spot_rl/envs/gaze_env.py

```diff
@@ -147,9 +241,32 @@ def get_success(self, observations):
 
 if __name__ == "__main__":
     spot = Spot("RealGazeEnv")
-    parser = get_default_parser()
-    parser.add_argument("--target-object", "-t")
-    args = parser.parse_args()
-    config = construct_config(opts=args.opts)
+    args = parse_arguments()
+    config = construct_config_for_gaze(opts=args.opts)
+    max_episode_steps = (
+        args.max_episode_steps
+        if args.max_episode_steps is not None
+        else config.MAX_EPISODE_STEPS
+    )
+    config = update_config_for_multiple_gaze(
```

Comment:
The reason why we are also calling `update_config_for_multiple_gaze` is because the file `gaze_env.py` by itself (through CLI) can also be used for running the multi-gaze policy.

We are allowing the user to feed in 
1. List of gaze_targets
2. Flag - should each gaze_target be picked up
3. Int - Max limit on number of steps.

Thus taking that input, we can alter or modify the config and perform single object gaze or multi-gaze.

Thus we are calling the `update_config_for_multiple_gaze` method here.

Not sure if it is the best idea to absorb it inside `construct_config_for_gaze()`. The idea for `construct_config_for_gaze()` method is to create the basic config generic enough for a gaze policy (we have followed same approach for nav & pick too) .. without need/dependence on additional params/arguments.

Reply:
combine these functions and parameter overwrites

Reply:
Combined all of them inside `construct_config_for_gaze()` as arguments